coordinator/src/orderbook/db/orders.rs

```diff
@@ -111,12 +114,19 @@ impl From<OrderbookNewOrder> for NewOrder {
                 .to_f32()
                 .expect("To be able to convert decimal to f32"),
             order_type: value.order_type.into(),
+            expiry: value.expiry,
         }
     }
 }
 
-pub fn all(conn: &mut PgConnection) -> QueryResult<Vec<OrderbookOrder>> {
-    let orders: Vec<Order> = orders::dsl::orders.load::<Order>(conn)?;
+pub fn all(conn: &mut PgConnection, show_expired: bool) -> QueryResult<Vec<OrderbookOrder>> {
```

Comment:
perhaps consider an enum `ShowExpired{Yes,No}` instead of the bool? It would be much easier to read & maintain, and is super easy to roll out.